.github/workflows/ci.yml

```diff
@@ -1,26 +1,18 @@
 name: ci-pipeline
-
 on:
   push:
     branches: [ master ]
   pull_request:
     branches: [ master ]
-
 jobs:
-
   ci:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        go: [ '1.19', '1.x' ]
```

Comment:
stable / old-stable would dynamically match both supported versions of go and not need updating, right?